data/strings.yml

```diff
@@ -156,17 +156,21 @@ code_signing:
         is_expose: Expose for Pull Request
     prov_profile_details_popup:
       menus:
-        matching_certificates: Matching Certificates
+        matching_certificates: Included Certificates
         devices: Devices
         capabilities: Capabilities
       device_registration_details:
-          not_registered: Not registered on bitrise
-          registered: Registered on Bitrise by
+        not_registered: Not registered on Bitrise
+        registered: Registered on Bitrise by
+      matching_certificates_details:
+        missing_or_expired_notice: "There is no matching certificate right now. They are either missing or expired.<br><strong>Upload at least one certificate that is matching with one of your provisioning profile.</strong>"
```

Comment:
I would separate these to `missing_or_expired_notice_part_1` and `missing_or_expired_notice_part_2` and put them in separate paragraphs in the template.